src/main/preload/mattermost.js

```diff
@@ -68,6 +68,32 @@ document.addEventListener('mouseup', () => {
   ipcRenderer.sendToHost('mouse-up');
 });
 
+const parentTag = (target) => {
+  if (target.parentNode) {
+    return target.parentNode.tagName.toUpperCase();
+  }
+  return null;
+};
+
+document.addEventListener('mouseover', (event) => {
```

Comment:
because I didn't have enough coffee and I looked for the event in the browserView and missed completely the webcontents one. I have reworked the code to use that one, much simpler and effective. Thanks!